seq2science/rules/quantification.smk

```diff
@@ -1,6 +1,6 @@
 from seq2science.util import get_bustools_rid
-
-
+import os
+import glob
```

Comment:
the general order in this is

1. stdlib
2. packages
3. current/local package

This would mean:

```
import os
import glob

from seq2science.util import get_bustools_rid
```